### PR TITLE
[WAF] Create OWASP section

### PR DESCRIPTION
**Pull request by pedrosousa**
Converts the existing OWASP page into a section, given the current page size.

### CODE REVIEW

File: content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+weight: 2
+meta:
+  title: OWASP ruleset concepts
+---
+
+# OWASP ruleset concepts
+
+## Paranoia level
+
+The paranoia level (PL) classifies OWASP rules according to their agressiveness. Paranoia levels vary from PL1 to PL4, where PL4 is the most strict level.
```
Comment (Maddy-Cloudflare) [inline content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md]:
```suggestion
The paranoia level (PL) classifies OWASP rules according to their aggressiveness. Paranoia levels vary from PL1 to PL4, where PL4 is the most strict level.
```

File: content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Each rule in the OWASP managed ruleset is associated with a paranoia level. Rules associated with higher paranoia levels are considered more agressive and provide increased protection. However, they might cause more legitimate traffic to get blocked due to false positives.
```
Comment (Maddy-Cloudflare) [inline content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md]:
```suggestion
Each rule in the OWASP managed ruleset is associated with a paranoia level. Rules associated with higher paranoia levels are considered more aggressive and provide increased protection. However, they might cause more legitimate traffic to get blocked due to false positives.
```

File: content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+When you configure the paranoia level of the OWASP ruleset you are enabling all the rules belonging to all paranoia levels up to the level you select. For example, if you configure the ruleset paranoia level to PL3, you are enabling rules belonging to paranoia levels PL1, PL2, and PL3.
```
Comment (Maddy-Cloudflare) [inline content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md]:
```suggestion
When you configure the paranoia level of the OWASP ruleset, you are enabling all the rules belonging to all paranoia levels up to the level you select. For example, if you configure the ruleset paranoia level to PL3, you are enabling rules belonging to paranoia levels PL1, PL2, and PL3.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md (hunk 1 above)
Comment (pedrosousa) [reply content/waf/managed-rules/reference/owasp-core-ruleset/concepts.md]: Oops... thanks!

**Pull request merged**